src/bzfs/bzfs.cxx

```diff
@@ -7645,7 +7645,8 @@ int main(int argc, char **argv)
                     unsigned char ubuf[MaxPacketLen];
                     bool     udpLinkRequest;
                     // interface to the UDP Receive routines
-                    int id = NetHandler::udpReceive((char *) ubuf, &uaddr,
+                    int buffLen;
```

Comment:
an unpleasant aspect of by-reference return values in functions is that it encourages this style of uninitialized declaration. Sure, the value will be set in the next call, but the next reader of this code might not know that. At a minimum, I would initialize all variables to reasonable defaults.